sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -39,11 +39,11 @@ public final class StorageBlobClient: PipelineClient {
     /// API version of the Azure Storage Blob service to invoke. Defaults to the latest.
     public enum ApiVersion: String {
         /// API version "2019-02-02"
-        case v20190202 = "2019-02-02"
+        case v20200210 = "2020-02-10"
```

Comment:
Shouldn't we be adding them here rather than replacing?

Reply:
Eventually, probably, but since we don't even have a first preview out, it doesn't make a lot of sense to have a bunch of conditional logic in place to support multi-API scenarios. We haven't even brainstormed what that would look like for either platform.